### PR TITLE
Add client and talk room fields to event

### PR DESCRIPTION
**Pull request by oliveiraigorm**
image
image

### CODE REVIEW

File: src/components/Editor/Properties/PropertyTalkButton.vue [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<template>
+	<button v-if="!isSlot && !isReadOnly"
```
Comment (frozenbanana) [inline src/components/Editor/Properties/PropertyTalkButton.vue:2]: same

File: src/components/Editor/Properties/PropertyTalkButton.vue [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+			if (doesDescriptionContainTalkLink(this.calendarObjectInstance.description)) {
+				return true
+			}
+
+			return false
+		},
+	},
+	methods: {
+		async createTalkRoom() {
+			const NEW_LINE = '\r\n'
```
Comment (frozenbanana) [inline src/components/Editor/Properties/PropertyTalkButton.vue:44]: the NEW_LINE variable should be moved to one place. Perhaps Utils

File: src/components/Editor/Properties/PropertyClientPicker.vue [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+	},
+}
+</script>
+
+<style scoped>
+.talk-button{
+	margin-top: 0.3rem !important;
+	background-color: white !important;
+	border-radius: 6px !important;
+	border-color: var(--color-main-text) !important;
```
Comment (frozenbanana) [inline src/components/Editor/Properties/PropertyClientPicker.vue:109]: is `!important` really necessary?

**Review by frozenbanana — DISMISSED** [review]
We have used semi-colons everywhere on adminly so I am confused why the linting has not catched it. Lets stick to semi-colons.

**Comment by oliveiraigorm** [comment]
> We have used semi-colons everywhere on adminly so I am confused why the linting has not catched it. Lets stick to semi-colons.

I guess lint here is not quite well configured since it was forked, maybe @mwalbeck has some input on that to make it more organized

**Review by mwalbeck — CHANGES_REQUESTED** [review]
If you add a video call link before adding a client, then when you add the client, the video call link gets removed.

Also can we get the select client to also show here:
image

I think it'll create some confusion if you have selected a client, and it keeps saying "Select client"

**Comment by oliveiraigorm** [comment]
> If you add a video call link before adding a client, then when you add the client, the video call link gets removed.
> 
> Also can we get the select client to also show here: image
> 
> I think it'll create some confusion if you have selected a client, and it keeps saying "Select client"

Did some changes improving it, with an add and one remove button. The styling of them is not complete and can be done on the other branch.

About the error I couldn't reproduce it, can you try again and tell me the specific steps?

**Comment by mwalbeck** [comment]
The functionality is exactly what I was looking for. Now it's just the styling, and I think it should be an `x` instead of a `-` but do you want to deal with that in the other branch?

**Comment by oliveiraigorm** [comment]
> The functionality is exactly what I was looking for. Now it's just the styling, and I think it should be an `x` instead of a `-` but do you want to deal with that in the other branch?

Yes, exactly. There's already a X SVG over there so that's what I had in mind

**Review by mwalbeck — APPROVED** [review]

**Pull request merged**